extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -24,6 +24,10 @@ namespace Microsoft.Windows.DevHome.SDK
         [experimental]
         [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
         QuickStartProject = 5,
+
+        [experimental]
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 5)] 
+        SourceControlIntegration = 6,
```

Comment:
The SDK change from this PR is temporary and will be removed once [https://github.com/microsoft/devhome/pull/3206](url) is merged.